### ISSUE TITLE
Can't compile code in Windows

### ISSUE THREAD

**Issue by somnuk-wk** [fsharp/fsharpbinding#1001]
In Emacs 24.4.1
When I execute command ```C-c C-c```

```
Compilation started at Tue May  5 11:22:11

c:/Program Files/Microsoft SDKs/F#/3.1/Framework/v4.0/fsc.exe --nologo e:/source/fsharp/test-syntax/Hello.fs
'c:/Program' is not recognized as an internal or external command,
operable program or batch file.

Compilation exited abnormally with code 1 at Tue May  5 11:22:11
```
Look like we need to put quotes around binary path somewhere.

**Comment by rneatherway** [comment]
I'm pretty sure this is the offending code: https://github.com/fsharp/fsharpbinding/blob/master/emacs/fsharp-mode.el#L293

And it looks as though the other cases will have the same problem. I guess people don't use that function very often! A PR would be very welcome, or I will get to this in a bit.

**Comment by somnuk-wk** [comment]
I just fixed the line you point and it work on my machine.

```  ((equal ext "fs")  (concat "\"" fsharp-compile-command "\"" " --nologo " file))```

Thank you.

**Comment by rneatherway** [comment]
I was just having a quick look and I think the more robust fix would be to replace `concat` with `combine-and-quote-strings`. Would you mind trying the line:

    ((equal ext "fs")  (combine-and-quote-strings (list fsharp-compile-command "--nologo" file)))

instead?

**Comment by somnuk-wk** [comment]
All right, I tries you suggestion and it work well.
This's the first time I've edited lisp code.

Thanks

**Comment by rneatherway** [comment]
Great, thanks for trying! I will fix this and roll it into the next release.

**Comment by rneatherway** [comment]
Fixed in #1005
